docs/apps/interchain-accounts/development.md

```diff
@@ -1,36 +1,36 @@
 <!--
-order: 5
+order: 2
 -->
 
 # Development use cases
 
-The initial version of interchain accounts allowed for the controller module to be extended by providing it with an underlying application which would handle all packet callbacks.
+The initial version of Interchain Accounts allowed for the controller submodule to be extended by providing it with an underlying application which would handle all packet callbacks.
 That functionality is now being deprecated in favor of alternative approaches. 
 This document will outline potential use cases and redirect each use case to the appropriate documentation. 
 
 ## Custom authentication 
 
 Interchain accounts may be associated with alternative types of authentication relative to the traditional public/private key signing. 
-If you wish to develop or use interchain accounts with a custom authentication module, we recommend you use ibc-go v6 or greater. 
+If you wish to develop or use Interchain Accounts with a custom authentication module and do not need to execute custom logic on the packet callbacks, we recommend you use ibc-go v6 or greater and that your custom authentication module interacts with the controller submodule via the [`MsgServer`](./messages.md).
 
-The custom authentication module should interact with the controller module via the [MsgServer](./messages.md).
+If you wish to consume and execute custom logic in the packet callbacks, then please read the section [Packet callbacks](#packet-callbacks) below.
 
 ## Redirection to a smart contract
 
 It may be desirable to allow smart contracts to control an interchain account.
-To faciliate such an action, the controller module may be provided an underlying application which redirects to smart contract callers. 
+To faciliate such an action, the controller submodule may be provided an underlying application which redirects to smart contract callers. 
 An improved design has been suggested in [ADR 008](https://github.com/cosmos/ibc-go/pull/1976) which performs this action via middleware. 
 
 Implementors of this use case are recommended to follow the ADR 008 approach.
 The underlying application may continue to be used as a short term solution for ADR 008 and the [legacy API](./auth-modules.md#registerinterchainaccount) should continue to be utilized in such situations. 
 
 ## Packet callbacks
 
-If a developer requires access to packet callbacks for their use case, then they having the following options:
+If a developer requires access to packet callbacks for their use case, then they have the following options:
```

Comment:
What if a dev requires access to packet callbacks, but does not have Cosmwasm or any other smart contract capability enabled on the chain? What happens in this case when the legacy api is no longer available?

Reply:
I believe we will not deprecate the legacy API until that use case is covered by using the `MsgServer` + ADR-008.